clinica/iotools/converters/aibl_to_bids/aibl_utils.py

```diff
@@ -520,27 +521,27 @@ def create_file(image, modality, bids_dir, overwrite):
         cprint(
             msg=(
                 f"[{modality.upper()}] Processing subject {subject} "
-                f"in session {session}"
+                f"in session {session_id}"
             ),
             lvl="info",
         )
 
-    session = viscode_to_session(session)
+    session_id = viscode_to_session(session_id)
```

Comment:
This part is confusing. I'd expect something like

```suggestion
    session_id = viscode_to_session_id(viscode)
```